### PR TITLE
Add footer 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- The footer consists of our template. #8 

**What is the new behavior (if this is a feature change)?**
- The quality and content of the footer has been updated. #8

### CODE REVIEW

File: src/components/molecules/Footer/index.jsx [hunk 1]
```diff
@@ -2,68 +2,85 @@
...
-                        </li>
-                        </ul>
-                    </MDBCol>
+                        <MDBCol md="4">
+                            <h5 className="title">Dienste</h5>
+                            <hr className="agency-red mb-4 mt-0 d-inline-block" />
+                            <ul>
+                            <li className="list-unstyled">
+                                <Link to="/service/online-presence">
+                                <   MDBIcon icon="globe" />Shop
```
Comment (kleberbaum) [inline src/components/molecules/Footer/index.jsx]: whitespace?

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/components/molecules/Footer/index.jsx (hunk 1 above)
Comment (Aichnerc) [reply src/components/molecules/Footer/index.jsx]: I ◽◽◽◽◽◽ really ◽◽◽◽◽◽ like ◽◽◽◽◽◽ whitespaces.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**